woocommerce/google-listings-and-ads
# PMax Assets: Support the portrait marketing images

### Changes proposed in this Pull Request:

This PR implements the "Assets form" part of 📌 [Assets form and related components](https://github.com/woocommerce/google-listings-and-ads/issues/1787#fe-misc) in #1787.

- Support the portrait marketing image and adjust the maximum number of each marketing image type to make the sum of the maximum limits 20.

The limits are adjusted to:
- MARKETING_IMAGE: min 1, max 8
- SQUARE_MARKETING_IMAGE: min 1, max 8
- PORTRAIT_MARKETING_IMAGE: min 0, max 4
   - Image size: (4:5) 960 x 1200 recommended; 480 x 600 min

### Screenshots:

#### 📷 Added portrait marketing image asset and adjusted maximum limits

![image](https://user-images.githubusercontent.com/17420811/217432622-1bb30939-1516-4e3a-8eff-4956081c4ab4.png)

#### 📷 Validation of the portrait marketing image asset

![image](https://user-images.githubusercontent.com/17420811/217432651-e9fea07b-6b2f-45dc-81e7-1a8d30c44551.png)

### Detailed test instructions:

1. Go to the campaign creation or editing page.
2. Check if the limits of each marketing image type match the proposed solution.
3. Check if the UI of the portrait marketing image asset work well.
4. Click the "Save changes" button to see if the validation of all marketing image types works correctly.

### Additional details:

- Reason for adjustment: pcTzPl-12o-p2#comment-2228
- Adopted solution for the present: pcTzPl-12o-p2#comment-2247

### Changelog entry
